net/haproxy/Makefile

```diff
@@ -28,7 +28,7 @@ DPCRE_DESC=		Link dynamically
 SPCRE_DESC=		Link statically
 DEVICEATLAS_DESC=	DeviceAtlas Device Detection support
 PROMEX_DESC=		Enable Prometheus exporter
-OPTIONS_DEFAULT=	SPCRE OPENSSL
+OPTIONS_DEFAULT=	SPCRE OPENSSL PROMEX
```

Comment:
Don't change the default options. Add/change the relevant lines in the pfSense repository in https://github.com/pfsense/pfsense/blob/master/tools/conf/pfPorts/make.conf

Reply:
done: https://github.com/pfsense/pfsense/pull/4320